api/integrations/lead_tracking/hubspot/client.py

```diff
@@ -64,8 +64,8 @@ def create_contact(self, user: FFAdminUser, hubspot_company_id: str) -> dict:
         )
         return response.to_dict()
 
-    def create_company(self, name: str) -> dict:
-        properties = {"name": name}
+    def create_company(self, name: str, organisation_id: int) -> dict:
+        properties = {"name": name, "orgid": str(organisation_id)}
```

Comment:
Minor: I'd love to see the hubspot parameters pulled out as constants in e.g. `hubspot.constants`.